packages/gestalt/src/Fieldset.js

```diff
@@ -0,0 +1,34 @@
+// @flow strict
+import type { Node } from 'react';
+import cx from 'classnames';
+import Text from './Text.js';
+import labelStyles from './Label.css';
+import formStyles from './FormElement.css';
+import formLabelStyles from './FormLabel.css';
+import boxWhitespaceStyles from './boxWhitespace.css';
+import whitespaceStyles from './Whitespace.css';
+import boxStyles from './Box.css';
+
+type Props = {|
+  children: Node,
+  legend: string,
+  legendDisplay?: 'visible' | 'hidden',
+|};
+
+export default function Fieldset({ legend, legendDisplay = 'visible', children }: Props): Node {
+  return (
+    <fieldset className={cx(formStyles.unstyled, whitespaceStyles.p0, whitespaceStyles.m0)}>
+      <legend
+        className={cx(
```

Comment:
can we do here `classnames` to be more consistent? 
![image](https://user-images.githubusercontent.com/10593890/117056594-7dede700-acea-11eb-83b8-73055acfd5fb.png)